docs/components/cards/cards.css

```diff
@@ -43,6 +43,7 @@
     top: 2px;
     right: 5px;
     color: var(--color-red);
+    padding: 5px;
```

Comment:
you could have this in relative unit like rem or em. But not a breaking change as of now

Reply:
values are very small so i am using px here. Thank you for the suggestion